README.md

```diff
@@ -224,6 +224,82 @@ permanent: true
 The permanent and runtime settings are independent, so you can set only the runtime, or only the permanent.  You cannot
 set both permanent and runtime to `false`.
 
+Examples of Options
+-------------------
+
+Permit TCP traffic for port 80 in permanent default zone:
+
+```yaml
+firewall:
```

Comment:
The firewall role `firewall` variable value is a list - so this should be
```yaml
firewall:
  - port: 80/tcp
    state: enabled
```

I note that there are some corrections we need to make in the role README as well - like https://github.com/linux-system-roles/firewall#state
https://github.com/linux-system-roles/firewall#example-playbooks
and in "It is also possible to combine several settings into blocks:" there are a couple of extraneous dashes `-` in the left hand column.

Also, the README https://github.com/linux-system-roles/firewall#variables section should start off with something like 
```
The firewall role uses the variable `firewall` to specify the parameters.  This variable is a `list` of `dict` values.  Each `dict` value is comprised of one or more keys listed below.
```